src/cmd/tools/read_data_files/main/main.go

```
@@ -86,6 +87,14 @@ func main() {
 	}
 	log := rawLogger.Sugar()
 
+	filterIds := strings.Split(*idFilter, ",")
+	filterIdRegex, err := regexp.Compile(strings.Join(filterIds, "|"))
```

Use `MustCompile()` to fail early and loudly.